### PR TITLE
Test csvReader by Nicolas and DamienD

### PR DESCRIPTION
**Pull request by DuboisD**
Programme de lecture de fichiers csv ainsi que les tests qui l'accompagnent.

### CODE REVIEW

**Comment by Tony07200** [comment]
No description for this PR ...

File: java/src/ReaderCSV.java [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    // private final static String RESSOURCE_PATH = "data/";
+    // private final static String COUNTRY_FILE_NAME = "country1.csv";
+    private final static char SEPARATOR = ',';
+
+    public List<Country> findCoutries(String RESSOURCE_PATH, String COUNTRY_FILE_NAME, boolean header) throws IOException {
+
+        File file = new File(RESSOURCE_PATH + COUNTRY_FILE_NAME);
+        FileReader fr = new FileReader(file);
+
+        com.opencsv.CSVReader csvReader = new com.opencsv.CSVReader(fr, SEPARATOR);
```
Comment (mbaechler) [inline java/src/ReaderCSV.java]: please implement you own CSV parser

File: java/src/ReaderCSV.java [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+        String[] nextLine = null;
+        List<Country> countries = new ArrayList<>();
+
+        // List<String> data = new ArrayList<String[]>();
+
+        if (header) {
```
Comment (Tony07200) [inline java/src/ReaderCSV.java]: What is header ? Where is it set ?

File: java/test/Test1.java [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+        assertThat(result).extracting("name").doesNotContain("");
+
+        //Assert.assertTrue(result.get(1).getName().equals("Belgique"));
+        //Assert.assertTrue(result.get(0).getName().equals("France") && result.get(0).getIndicateur().equals("FR"));
+        //Assert.assertTrue(result.size() == 3);
+
+    }
+
+    @Test
+    public void ShouldReadCSVWithSpaces() throws IOException {
```
Comment (Tony07200) [inline java/test/Test1.java]: Why check size when test run for spaces ?

Comment (NicolasMtz26) [reply java/test/Test1.java]: We keep precedent check for all tests and adding new verification.
The new thing is that we had complete the second "assertThat" with the entire content of the List that we expected to be sure that France is register and not France with spaces

File: java/src/ReaderCSV.java (hunk 2 above)
Comment (Tony07200) [reply java/src/ReaderCSV.java]: Ok , i didn't saw.. thx

File: java/test/Test1.java (hunk 3 above)
Comment (Tony07200) [reply java/test/Test1.java]: Ok good

**Pull request closed without merging**